docs/en/contributor/Contributor-Tools.md

```diff
@@ -227,26 +212,17 @@ $ mvn clean install -Pgenerate
 ```
 
 ## Usage of `./bin/alluxio`
+ 
+Please refer to [Alluxio commands]({{ '/en/operation/User-CLI.html' | relativize_url }})
+for all available commands.
+
+Some commands have different prerequisites.
 
-Most commands in `bin/alluxio` are for developers. The following table explains the description and
-the syntax of each command.
-
-<table class="table table-striped">
-    <tr><th>Command</th><th>Args</th><th>Description</th></tr>
-    {% for dscp in site.data.table.developer-tips %}
-        <tr>
-            <td>{{dscp.command}}</td>
-            <td>{{dscp.args}}</td>
-            <td>{{site.data.table.en.developer-tips[dscp.command]}}</td>
-        </tr>
-    {% endfor %}
-</table>
-
-In addition, these commands have different prerequisites. The prerequisite for the `format`,
-`formatWorker`, `journalCrashTest`, `readJournal`, `runClass`, `version`, `validateConf` and `validateEnv` commands is
-that you have already built Alluxio (see
-[Build Alluxio Master Branch]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}) about how to
-build Alluxio manually).
-Further, the prerequisite for the `fs`, `logLevel`, `runTest` and `runTests` commands is that
-the Alluxio service is up and running.
+The prerequisite for the `format`, `formatMaster`, `formatWorker`, `getConf`, `runClass`, `runHmsTests`, `runHdfsMountTests`, 
```

Comment:
when i read this entire section on `Usage of ./bin/alluxio`, i don't feel it's too useful in its current state. we list a few commands and don't give much context on how they should be used. i would rather see this section contain a reference to the user-cli docs and then show an example workflow to test/validate a cluster. for example, an outline for this section could be like:
- developers can use the `bin/alluxio` command to manage and test your local Alluxio cluster
- see the full list of commands in user cli docs
- assuming alluxio is properly configured for a local cluster (link to local cluster setup), an example usage of the bin/alluxio script:
  - (not sure if exists) run validateEnv/validateConf for local cluster to validate your local setup
  - run format to clear metadata and worker storage
  - start a local alluxio cluster (dunno cmd)
  - run fsadmin report to check cluster status summary
  - run runTests to execute read/write operations through the local cluster to the configured UFS
  - run fs ls / to see test files
  - dunno if there are other commands to test on a running cluster

Reply:
Or maybe when we describe the command in User-CLI, we talked about whether it needs to be used in a running cluster

Reply:
On Rico's comment for the new section, I think the new section surely adds useful information, but should not be in the `Contributor Tools` page. I would do this in 2 steps:
1. Add info according to Lu's suggestion, state if the command needs Alluxio cluster to be up in user CLI page. And in the contributor tool page, refer to that.
2. (Another PR to-be) add a section about how validation commands work together, maybe in a separate page or in the User CLI page.

Reply:
sounds good to me

Reply:
Added.